src/components/assets/Account.vue

```diff
@@ -275,6 +276,8 @@ export default defineComponent({
 
     const currentNetworkName = ref<string>(providerEndpoints[currentNetworkIdx.value].displayName);
 
+    const { isDappStakingV3 } = useDappStaking();
```

Comment:
Let's remove this if you are not using at this file

Reply:
@impelcrypto oh, this again. thank you. I've removed it in [e02e1de](https://github.com/AstarNetwork/astar-apps/pull/1150/commits/e02e1def91291bf9ed33cbfa7f2e5b9c65dce0e4)